### PR TITLE
More work to get it production ready

### PR DESCRIPTION
**Pull request by arkilic**

### CODE REVIEW

File: analysisstore/client/commands.py [hunk 1]
```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```
Comment (klauer) [inline analysisstore/client/commands.py]: Separating host/port may be more user-friendly

Comment (arkilic) [inline analysisstore/client/commands.py]: I have them separated in all other services. I was hoping we can move towards Olog-like port agnostic address model. I can reverse this for the time being.

Comment (klauer) [inline analysisstore/client/commands.py]: You could make it smart and look for a colon to split host/port, but that may make it needlessly complex

Comment (arkilic) [inline analysisstore/client/commands.py]: I did smth like this for the archiver a while back. It doesn't get too complex, worth the effort.

Comment (cowanml) [inline analysisstore/client/commands.py]: We should have a standardized nsls2 service config interface so
they're all the same and we have one place to change if we want to
switch something, like the host+port together or separate, and don't
have to keep rewriting the same thing.

-matt

Comment (arkilic) [inline analysisstore/client/commands.py]: We already have a standard method (at least for my services). This was my attempt to try something new and a little more refined. I am glad it was able to start some discussion. 
All in all, if I deviate from the standard and find out the new method is superior, I will be switching all configurations to the same method. RIght now, everything but this one follows the same standard

**Comment by hhslepicka** [comment]
Looks good to me! :+1:

**Pull request merged**